spec/unit/index.spec.ts

```diff
@@ -78,8 +79,10 @@ describe("ssim", () => {
   });
 
   test("should produce the right SSIM for bmp images as well (avion)", () => {
-    const { mssim } = lib.ssim(samples.avion, samples.avion_j2000_r1);
+    let { mssim } = lib.ssim(samples.avion, samples.avion_j2000_r1, {ssim: "fast"});
     expect(roundTo(mssim, 5)).toEqual(0.98078);
+    mssim = lib.ssim(samples.avion, samples.avion_j2000_r1).mssim;
+    expect(roundTo(mssim, 4)).toEqual(roundTo(0.98078,4));
```

Comment:
Similar to the main comment, and not something we need to tackle here but it may be time to come up with a consistent approach to rounding / tolerance and apply it consistently for the tests

Reply:
Agreed.